### ISSUE TITLE
Documentation : k3s edge cluster only supported on amd64 and arm64

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/open-horizon.github.io#278]
### What is the current documentation state?

The documentation is not clear about what k3s operating systems are supported.  k3s requires amd64 and arm64
A user tried to install Open Horizon edge cluster with k3s on armhf Raspberry Pi arm32.

Improve the upstream Open Horizon docs at `open-horizon.github.io/docs/installing/preparing_edge_cluster.md` 
which renders at https://open-horizon.github.io/docs/installing/preparing_edge_cluster.html

**Comment by joewxboy** [comment]
@Polber Can you either give us a little assistant with the info we need, or point us to someone to work with?

**Comment by johnwalicki** [comment]
@Polber will incorporate a pre-reqs section in this working document about Edge Cluster installation steps:
https://wiki.lfedge.org/display/OH/How+to+Set-up+and+Install+an+Edge+Cluster+Agent

Once the wiki page is complete, we'll re-merge it back into the main Open Horizon documentation webpage.

**Comment by joewxboy** [comment]
@Polber Are you still working on this issue, or can I re-assign to someone else?  Thanks.

**Comment by Polber** [comment]
@joewxboy No, I am no longer working on this. I went ahead and unassigned myself.

**Comment by joewxboy** [comment]
Closing
